atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbConstants.java

```diff
@@ -33,6 +33,7 @@
     public static final TableReference SCRUB_TABLE = TableReference.createWithEmptyNamespace("_scrub");
     public static final TableReference NAMESPACE_TABLE = TableReference.createWithEmptyNamespace("_namespace");
     public static final TableReference TIMESTAMP_TABLE = TableReference.createWithEmptyNamespace("_timestamp");
+    public static final TableReference ORACLE_TIMESTAMP_TABLE = TableReference.createWithEmptyNamespace("atlasdb_timestamp");
```

Comment:
why is this a specialty string and not prefixed? seems like we would want this to be consistent with the other tables with have a `a_` prefix